package.json

```diff
@@ -2,27 +2,24 @@
   "name": "ui-kit",
   "private": true,
   "scripts": {
-    "postinstall": "husky install & patch-package && npm run link:angular -w @coveo/atomic-angular-samples",
-    "start": "lerna run --parallel start",
-    "dev": "npm run start -- --scope @coveo/headless --scope @coveo/atomic",
-    "build": "lerna run clean && lerna run build",
-    "test": "lerna run --parallel test",
-    "test:watch": "lerna run --parallel test:watch --no-prefix",
-    "lint:check": "eslint .",
-    "lint:fix": "eslint --fix .",
+    "postinstall": "husky install && patch-package && npm run link:angular -w @coveo/atomic-angular-samples",
+    "dev:atomic": "concurrently \"npm run -w @coveo/headless dev\" \"npm run -w @coveo/atomic dev\"",
+    "build": "nx run-many --target=build",
+    "test": "nx run-many --target=test",
+    "e2e": "nx run-many --target=e2e",
+    "lint:check": "npx eslint .",
+    "lint:fix": "npx eslint --fix .",
```

Comment:
Ideally, those would be run at a package-level, but this is too big of a change for this PR, I'll do it separately.